src/eradius_server_sup.erl

```diff
@@ -18,11 +18,18 @@ start_link() ->
 
 start_instance(_ServerAddr = {ServerName, {IP, Port}}) ->
     ?LOG(info, "Starting RADIUS Listener at ~s", [printable_peer(IP, Port)]),
-    supervisor:start_child(?SERVER, [ServerName, IP, Port]).
+    supervisor:start_child(?SERVER, [ServerName, IP, Port]);
```

Comment:
Do we need in this `start_instance` clause at all? If I understand correctly according to https://github.com/travelping/eradius/pull/215/files#diff-7fa5d8722d6a9a3934327aa17f94d766a23aed20a907fca4d98c7764fbf7f785R158 it will be called with `Opts` every time (specified in configuration or `[]` if it is not)

Reply:
you are right. I left it because I don't want to break the api. In particular I don't know if exists some code that call start_instance without passing for the monitoring system

Reply:
yes, I see :+1:  So let's leave it as is for now. That could be adjusted later